docs/snippets/common/storybook-main-ts-module-resolution.ts.mdx

```diff
@@ -10,12 +10,14 @@ const config: StorybookConfig = {
   framework: '@storybook/your-framework',
   stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
   webpackFinal: async (config) => {
-    config.resolve.plugins = [
-      ...(config.resolve.plugins || []),
-      new TsconfigPathsPlugin({
-        extensions: config.resolve.extensions,
-      }),
-    ];
+    if(config.resolve){
```

Comment:
@noviceGuru, mind adjusting this snippet's JavaScript version to factor in the same changes here for accuracy and consistency?